### ISSUE TITLE
Cannot create event because of bugged field

### ISSUE THREAD

**Issue by kzhe2059** [davish/obsidian-full-calendar#533]
### Please check that this issue hasn't been reported before.

- [X] I searched previous [Bug Reports](https://github.com/davish/obsidian-full-calendar/issues?q=is%3Aissue+is%3Aopen+label%3Abug) and didn't find any similar reports.

### Expected Behavior

Creating events works and you select which calendar they're in

### Current behaviour

Events can't be created as shown in this image
![image](https://github.com/davish/obsidian-full-calendar/assets/29979557/65f1fe94-2ed1-49af-8509-1845abb64fd3)

### JavaScript console output

There is no view toggle in settings

### Steps to reproduce

Create new event by clicking on date

### Calendar information

2 Caldav from same url
1 ICS

### Which Operating Systems are you using?

- [ ] macOS
- [ ] Windows
- [X] Linux
- [X] Android
- [ ] iOS

### Obsidian Version

1.5.3

### Full Calendar Plugin Version

0.10.7

### Checks

- [X] I have tried it with all other plugins disabled and the error still occurs.

**Comment by Blondwolf** [comment]
Hello, I was also blocked by this. This is currently only working on read for caldav.  You can't add or modify events from caldav calendars. I hope it will be added

**Comment by kzhe2059** [comment]
That's disappointing, as I can't use a calendar app without the ability to create events. Hopefully that will change.

**Comment by HighLibrarian** [comment]
i added an ical calendar as read only, but added a second calendar that i use for daily notes. 
now i can add events to my daily notes but still see the events from my personal calendar. perhaps not what you need but it's the next best thing for me.

**Comment by Blondwolf** [comment]
Well for my part, I was able to synchronize my notes via ical as well. With those modules:
- iCal
- APIRequest

So basically, you scan your vault with iCal and it generates a file, for my side .ics; Then you can PUSH your file to a CalDav server such as Radicale. I also use a custom module for triggering https://github.com/Blondwolf/obsidian-file-trigger and finally I use a NodeRED server to create a requestable API.

This is a bit cumbersome and can surely be optimized but at least it update my FullCalendar agenda automatically by pulling then this calendar from my CalDav server. Would be nice to have someone integrate them all together.

Cheers

**Comment by brianbrandson** [comment]
Version: 1.6.7
OS: Ubuntu 24.04

It does the same. I cannot select my CalDav calendar. And also cannot modify existing events as it gives me Read-only events cannot be modified.

**Comment by Blondwolf** [comment]
Yes it does the same. Except "just" that you can see your events in calendar module, synchronize it with your phone and any calendar app.

Also somehow with my method, it doesn't make sens to be able to modify your events from the events. Think about it: you're scanning your notes to generate events. You don't create event to generate notes. So even if you were able to modify them, it will be override by the next event generation.

Personnaly I use two calendar, 1 for syncing with my notes, 1 for simple events.
